FirebaseAuth/Sources/Auth/FIRAuth.m

```diff
@@ -1975,6 +1976,16 @@ - (BOOL)updateCurrentUser:(nullable FIRUser *)user
     [self possiblyPostAuthStateChangeNotification];
     return YES;
   }
+  if (user) {
+    if ((user.tenantID != nil && ![self.tenantID isEqualToString:user.tenantID]) ||
+        (self.tenantID != nil && ![user.tenantID isEqualToString:self.tenantID])) {
```

Comment:
Can this be simplified to `if ((user.tenantID || self.tenantID) && ![self.tenantID isEqualToString:user.tenantID])`?

Reply:
Yeah, looks better.